lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -76,19 +81,40 @@ def invalid_channel_integrations(requirements)
           return unless channel_integrations
           [].tap do |errors|
             if channel_integrations.size > 1
-              errors << ValidationError.new(:multiple_channel_integrations, count: channel_integrations.size)
```

Comment:
We don't need count anymore? not really sure what count is used for.

Reply:
Yeah good question. As far as I can tell, `count` gets used in the translations when they are formatted for example like:
```
- translation:
      key: "txt.apps.admin.error.app_build.duplicate_requirements.one"
      title: "App builder job: requirements file contains duplicate key error"
      value: "requirements.json contains a duplicate key: %{duplicate_keys}"
  - translation:
      key: "txt.apps.admin.error.app_build.duplicate_requirements.other"
      title: "App builder job: requirements file contains duplicate keys error"
      value: "requirements.json contains duplicate keys: %{duplicate_keys}"
```

Note the `.one` and `.other` suffixes. The `missing_required_fields` doesn't handle a case like this though, i.e:

```
- translation:
      key: "txt.apps.admin.error.app_build.missing_required_fields"
      title: "App builder job: required key missing in requirements, e.g. \"title\" is required in \"my_custom_email_target\""
      value: "Missing required fields in requirements.json: \"%{field}\" is required in \"%{identifier}\""
```

...so I don't _think_ it needs this param passed.